aea/crypto/cosmos.py

```diff
@@ -820,6 +820,45 @@ def get_contract_instance(
         # Instance object not available for cosmwasm
         return None
 
+    @staticmethod
+    def _execute_shell_command(command: List[str]) -> Any:
```

Comment:
Return type is not accurately described in signature; I think it should be `Dict[str, str]`?